src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -850,6 +878,33 @@ export class ServerView extends DirMixin(PolymerElement) {
     this.dispatchEvent(makePublicEvent('RemoveAccessKeyRequested', {accessKeyId: accessKey.id}));
   }
 
+  setServerTransferredData(totalBytes) {
+    this.totalInboundBytes = totalBytes;
+  }
+
+  updateAccessKeyRow(accessKeyId, fields) {
+    let newAccessKeyRow;
+    if (accessKeyId === MY_CONNECTION_USER_ID) {
+      newAccessKeyRow = Object.assign({}, this.get('myConnection'), fields);
+      this.set('myConnection', newAccessKeyRow);
+    }
```

Comment:
Return?

Reply:
I'm not sure where theis was supposed to be referring to based on Github's UI.  It's on _formatInboundBytesUnit, which now properly returns based on the new `Intl`-based formatting